app/models/validations/sales/sale_information_validations.rb

```diff
@@ -54,6 +54,20 @@ def validate_discounted_ownership_value(record)
     end
   end
 
+  def validate_outright_sale_value_matches_mortgage_plus_deposit(record)
+    return unless record.saledate && record.form.start_year_after_2024?
+    return unless record.outright_sale?
+    return unless record.mortgage_used? && record.mortgage
```

Comment:
N.B. If mortgageused is "no" or "don't know", we don't ask mortgage or deposit so no validation is needed.